webdriver-spec.html

```diff
@@ -434,11 +434,15 @@
     <!-- queue --> <li><dfn><a href=https://infra.spec.whatwg.org/#queues>queue</a></dfn>
    </ul>
 
- <dt>Page Visibility
+ <dt>Performance
```

Comment:
This doesn't seem like a correct change --- the very next line refers to the `Page Visibility Specification`.